foreman/data_refinery_foreman/surveyor/management/commands/surveyor_dispatcher.py

```diff
@@ -73,6 +73,9 @@ def queue_surveyor_for_accession(accession: str) -> None:
 
     # We don't actually send the job here, we just create it.
     # The foreman will pick it up and dispatch it when the time is appropriate.
+    return survey_job
+
+    return survey_job
```

Comment:
how did the linter not catch this?

Reply:
Hmm, I guess that's fine apparently? Not ideal....